### PR TITLE
Switch off docker image build

### LINKED ISSUE #70: The hive docker image is built even if `local_docker_image` is set to false
## Current behaviour
The hive docker image is built regardless of the value of `local_docker_image`. 

## Expected behaviour
Should not run [build-docker book](https://github.com/Skatteetaten/terraform-nomad-hive/blob/master/dev/ansible/05_build_docker_image.yml) if `local_docker_image=false`. This is simply to save time.

## How to reproduce?
Run `make dev` or `make up` with `local_docker_image=false`.

## Suggestion(s)/solution(s) [Optional]
Add a [check](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#id2) in the [ansible playbook](https://github.com/Skatteetaten/terraform-nomad-hive/blob/master/dev/ansible/05_build_docker_image.yml) that makes it only run if `local_docker_image=true`. 

## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### PR DESCRIPTION
**Pull request by zhenik**
## Closes/fixes/resolves issue(s)?
Partly resolve #70 

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone

### CODE REVIEW

File: dev/ansible/05_build_docker_image.yml [hunk 1]
```diff
@@ -15,6 +16,7 @@
       args:
         TEST_DOWNLOAD_BUILD_ARGUMENT: test-arg
     source: build
+  when: lookup('env', 'build_docker_image') | bool or false
```
Comment (claesgill) [inline dev/ansible/05_build_docker_image.yml:19]: What means `bool or false`?

Comment (zhenik) [reply dev/ansible/05_build_docker_image.yml:19]: This part make look up and cast result to Boolean `lookup('env', 'build_docker_image') | bool`, part with `or false` is like default value, equal to `|| false`

Comment (claesgill) [reply dev/ansible/05_build_docker_image.yml:19]: Aha! Thanks for the clarification.

**Review by claesgill — APPROVED** [review]
LGTM

**Review by dangernil — APPROVED** [review]

**Pull request merged**